Plugin.pm

```diff
@@ -1210,7 +1215,7 @@ sub _albumItem {
 	my $artist = $album->{artist}->{name} || '';
 	my $albumName = $album->{title} || '';
 	my $showYearWithAlbum = $prefs->get('showYearWithAlbum');
-	my $albumYear = $showYearWithAlbum ? $album->{year} || (localtime($album->{released_at}))[5] + 1900 || 0 : 0;
+	my $albumYear = $showYearWithAlbum ? $album->{year} || substr($album->{release_date_stream},0,4) || 0 : 0;
```

Comment:
Might be worth a `getYearFromReleaseDate()` function - you're doing the same conversion several times.

Reply:
I respectfully disagree, as these are all one-line changes and the usage is really a simple assignment now rather than a conversion. The overhead for adding a function call wouldn't be worth it.

Reply:
I count six occurrences. Had I (or you) made this a helper function you'd only have had to change it once. Next time you'll have to do the same thing six times again. Unless you've already missed an occurrence 😉. But let's not get stuck on such details.